simulate-event.js

```diff
@@ -1,20 +1,19 @@
-var extend = require('xtend')
 
 /**
  * Set some default options.
  *
  * @type {Object}
  */
 var eventOptions = {
-  UIEvent: function (el) {
+  UIEvent: function () {
     return {
-      view: el.ownerDocument.defaultView
+      view: document.defaultView
```

Comment:
Sorry, was some thinking to myself. I was thinking that, I suppose, it already works if you pass `{ view: x }` as the option since it's merged with this so the change seems reasonable 😄 I don't have a ton of time to review it until tonight now, so I was just making sure I mentioned it.